acceptancetests/clean_resources.py

```diff
@@ -35,15 +35,20 @@ def clean(args):
             instance_groups = dict(substrate.iter_instance_security_groups())
             non_instance_groups = dict((k, v) for k, v in all_groups.items()
                                        if k not in instance_groups)
-            unclean = substrate.delete_detached_interfaces(
-                non_instance_groups.keys())
-            logging.info('Unable to delete {} groups'.format(len(unclean)))
+            try:
+                unclean = substrate.delete_detached_interfaces(
+                    non_instance_groups.keys())
+                logging.info('Unable to delete {} groups'.format(len(unclean)))
+            except:
```

Comment:
Same comment here re: being explicit about what we're catching (and for the try/except below).